templates/todo_list/tags.html

```diff
@@ -0,0 +1,36 @@
+{% extends 'base.html' %}
+{% block content %}
+  <h1>
+    Tags
+    <a class="btn btn-primary link-to-page" href="{% url 'todo_list:tag-create' %}">Add</a>
+  </h1>
+
+  {% if tags %}
+    <table class="table">
+      <tr>
+        <th>Name</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+      {% for tag in tags %}
+        <tr>
+          <td>
+            {{ tag.name }}
+          </td>
+          <td>
+            <a href="{% url 'todo_list:tag-update' tag.id %}" class="btn btn-warning">
+              Update
+            </a>
+          </td>
+          <td>
+            <a href="{% url 'todo_list:tag-delete' tag.id %}" class="btn btn-danger">
+              Delete
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.